elements/rh-progress-stepper/rh-progress-stepper.css

```
@@ -6,29 +6,47 @@
 }
 
 :host {
-  --_step-line-color: var(--rh-color-border-strong);
-  --_current-step-color: var(--_rh-color-status-disabled, #6a6e73);
-  --_current-step-label-color: var(--rh-color-text-primary);
-  --_step-line-angle: -180deg; /* sets the line direction between steps to horizontal */
-  --_step-line-position-block: 0 100%;
-  --_step-line-position-inline: calc(calc(100% - var(--_step-icon-width)) * -1) 100%;
-  --_step-line-size-inline: calc(100% - var(--_step-icon-width));
-  --_step-line-size-block: 100%;
-  --_step-template-areas:
+  --_step-line-color: /* @summary Color of connecting lines between steps */
```

see previous comments re: not documenting `--_` props, documenting system tokens. A note about syntax: IIRC, cem will associate comments with the prop on the next line, not the one for which the comment is placed at the end of the line

```suggestion
  /* @summary Color of connecting lines between steps */
  --_step-line-color: 
```

if on the other hand you intended this comment to apply to `--rh-color-border-strong`, then you did it correctly. Note however that on this line for example, the comment is misplaced

```css
  --_step-line-angle: -180deg; /* Line direction between steps to horizontal */
```